redaxo/src/addons/install/lib/archive.php

```diff
@@ -12,13 +12,23 @@ public static function extract($archive, $dir, $basename = '')
         $dir = rtrim($dir, '/\\');
         rex_dir::delete($dir);
 
-        if (class_exists('ZipArchive')) {
+        if (!class_exists('ZipArchive')) {
+            $archive = 'phar://' . $archive . '/' . $basename;
+            return rex_dir::copy($archive, $dir);
+        }
+
+        $zip = new ZipArchive();
+        if (!$zip->open($archive)) {
+            return false;
+        }
+
+        try {
+            if ('' === $basename) {
+                return $zip->extractTo($dir);
+            }
```

Comment:
@staabm Diese Weiche ist der Hauptunterschied.
Also wenn $basename nicht gesetzt, dann direkt zum Ziel entpacken.
Ansonsten wie gehabt erst in ein `[...].temp`-Dir.

@aeberhard sagte ja im Issue, dass das bei ihm helfen würde, wenn der rename wegfällt.